sql/moz-fx-data-shared-prod/fenix_derived/firefox_android_clients_v1/checks.sql

```diff
@@ -0,0 +1,37 @@
+#fail
+{{ is_unique(columns=["client_id"]) }}
```

Comment:
If this check is expected to fail due to the  [known duplicates](https://mozilla-hub.atlassian.net/browse/DENG-656) so we add a message about the known issue or is it only running for new dates?

Reply:
This is not expected to fail. I deduplicated the table and hence if this check starts failing then it's an indication of something having gone wrong.